worker/caas/broker.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
```

Comment:
can you please give this package a more specific name? caasbroker perhaps?

Reply:
ok